docs/user/integrations.rst

```diff
@@ -44,7 +44,20 @@ All categories of documentation benefit from being *continuously* built and publ
 If you are managing your documentation in git,
 you will be able to preview changes and see accepted changes published immediately using :doc:`/pull-requests`.
 
+Automated versioning
+--------------------
+
+Maintaining your documentation in a git repository with Read the Docs' automated CI/CD pipeline
+also means that you will be able to align your project's **release cycle** with your documentation.
+Whenever you announce a new **version** for your documentation,
+you will probably tag that version in your git repository.
+Read the Docs automatically registers the change and adds a new version to your documentation.
```

Comment:
We don't always do this, so we should not overpromise here. Users have to configure this.

Reply:
Yes, should definitely emphasize that it's ultimately an option that has to be configured.